docs/en/migration.md

```diff
@@ -2,16 +2,105 @@
+        ),
+        num_views=[1, 1],
+        pipelines=[train_pipeline1, train_pipeline2],
+        prefetch=prefetch,
+    ),
+    val=...)
+```
+
+In **MMSelfSup 1.x**, we separate `train_dataloader`, `val_dataloader` to summarize information correspodingly.
+
+Here is an example of `train_dataloader`:
+
+```python
+train_dataloader = dict(
+    batch_size=32,
+    num_workers=4,
+    persistent_workers=True,
+    sampler=dict(type='DefaultSampler', shuffle=True),
+    dataset=dict(
+        type=dataset_type,
+        data_root=data_root,
+        ann_file='meta/train.txt',
+        data_prefix=dict(img_path='train/'),
+        pipeline=train_pipeline))
+val_dataloader = ...
+```
+
+Besides, we remove the key of `data_source` to keep the consistent pipeline format with other OpenMMLab projects. Please refer to [1_config.md](user_guides/1_config.md) for more details.
+
+#### Models
+
+In the config of models, there are two main different parts.
+
+One is that there is a new key called `data_preprocessor`, which is responsible for preprocessing the data, like normalization, channel conversion, etc.
+
+The other one is that a new key `loss` in `head` to determine the loss function of the algorithm.
+
+#### Schedules
+
+| MMSelfSup 0.x    | MMSelfSup 1.x   | Remark                                                                                                                          |
+| ---------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| optimizer_config | /               | It is removed.                                                                                                                  |
+| /                | optim_wrapper   | The `optim_wrapper` provides a common interface for updating parameters.                                                        |
+| lr_config        | param_scheduler | The `param_scheduler` is a list to set learning rate or other parameters, which is more flexible.                               |
+| runner           | train_cfg       | The loop setting (`EpochBasedTrainLoop`, `IterBasedTrainLoop`) in `train_cfg` controls the work flow of the algorithm training. |
+
+### Folders and Files
+
+The table below records the general modification of the folders and files.
+
+| MMSelfSup 0.x         | MMSelfSup 1.x       | Remark                                                                                                                                                   |
+| --------------------- | ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| apis                  | /                   | Currently, the `apis` folder has been removed, it might be added in the future.                                                                          |
```

Comment:
```suggestion
| apis                  | /                   | Currently, the `apis` folder has been **removed**, it might be added in the future.                                                                          |
```